src/client/components/footer/footer.styles.css

```diff
@@ -1,8 +1,9 @@
 footer {
   background-color: #f8f8f8;
-  position: fixed;
   width: 100%;
```

Comment:
Why did you remove `position: fixed`?

Reply:
to scroll up and down with the page...but i put it back